spectral_cube/dask_spectral_cube.py

```diff
@@ -182,21 +182,24 @@ class FilledArrayHandler:
     """
 
     def __init__(self, cube, fill=np.nan):
-        self._cube = cube
+        self._data = cube._data
+        self._mask = cube._mask
```

Comment:
@keflavich are these the changes you mean? These changes are required for dask.distributed to work - somehow storing a cube on these classes causes serialization issues.

Reply:
Yes; got it, thanks, that clarifies it.